framework/components/ASelect/ASelect.spec.js

```diff
@@ -18,6 +18,22 @@ context("ASelect", () => {
       });
   });
 
+  it("allows overflow styling", () => {
+    const playground = "#many-dropdown-options + .playground";
+    cy.get(`${playground} .a-select__selection`).eq(0).click()
+    cy.get(".a-select__menu-items")
```

Comment:
it may be worth checking also that the `.a-select__menu-items__wrapper` element has the `overflow-y-scroll` class added in this case